blueocean-indexing/src/main/java/io/jenkins/blueocean/indexing/Transformers.java

```diff
@@ -0,0 +1,170 @@
+                    public Container<BlueArtifact> getArtifacts() {
+                        return null;
+                    }
+
+                    @Override
+                    public BluePipelineNodeContainer getNodes() {
+                        return null;
+                    }
+
+                    @Override
+                    public Collection<BlueActionProxy> getActions() {
+                        return null;
+                    }
+
+                    @Override
+                    public BluePipelineStepContainer getSteps() {
+                        return null;
+                    }
+
+                    @Override
+                    public Object getLog() {
+                        return null;
+                    }
+
+                    @Override
+                    public BlueQueueItem replay() {
+                        throw new UnsupportedOperationException();
+                    }
+
+                    @Override
+                    public Link getLink() {
+                        return parent.rel("runs/"+getId());
+                    }
+                };
+            }
+        };
+    }
+
+    public static final Function<BlueRun, Document> RUN_TO_DOCUMENT = new Function<BlueRun, Document>() {
+        @Nullable
+        @Override
+        public Document apply(BlueRun input) {
+            Document document = new Document();
+            document.add(new LongField(BlueRun.ID, Integer.valueOf(input.getId()), Store.YES));
+            document.add(new NumericDocValuesField(sortField(BlueRun.ID), Integer.valueOf(input.getId())));
+            document.add(new StringField(BlueRun.RESULT, input.getResult().name(), Store.YES));
+            document.add(new StringField(BlueRun.ORGANIZATION, input.getOrganization(), Store.YES));
+            document.add(new StringField(BlueRun.PIPELINE, input.getPipeline(), Store.YES));
+            document.add(new StringField(BlueRun.TYPE, input.getType(), Store.YES));
+            document.add(new StringField(BlueRun.RUN_SUMMARY, input.getRunSummary(), Store.YES));
+            document.add(new LongField(BlueRun.START_TIME, input.getStartTime().getTime(), Store.YES));
+            document.add(new LongField(BlueRun.END_TIME, input.getEndTime().getTime(), Store.YES));
+            document.add(new LongField(BlueRun.ENQUEUE_TIME, input.getEnQueueTime().getTime(), Store.YES));
+            document.add(new LongField(BlueRun.DURATION_IN_MILLIS, input.getDurationInMillis(), Store.YES));
+            document.add(new LongField(BlueRun.ESTIMATED_DURATION_IN_MILLIS, input.getEstimatedDurtionInMillis(), Store.YES));
+            document.add(new NumericDocValuesField(sortField(BlueRun.START_TIME), input.getStartTime().getTime()));
+            document.add(new NumericDocValuesField(sortField(BlueRun.END_TIME), input.getEndTime().getTime()));
+            document.add(new NumericDocValuesField(sortField(BlueRun.ENQUEUE_TIME) + "_sort", input.getEnQueueTime().getTime()));
```

Comment:
🐛  Need to remove ` + "_sort"` on the next few lines as `sortField` does the prefixing